docs/guide/field_type_reference.md

```diff
@@ -1265,7 +1355,7 @@ The Value class of this Field Type contains the following properties:
 
 ###### Constructor
 
-The `MapLocation\Value` constructor will initialize a new Value object with the values provided. Two floats and a string are expected.
+The `MapLocation\Value` constructor will initialize a new Value object with three values provided: two floats and a string.
```

Comment:
This sentence may suggest that constructor accepts three argument with is not true. More precise version IMHO: 

> The `MapLocation\Value` constructor will initialize a new Value object with values provided as hash. Accepted keys are `latitude` (`float`), `longitude` (`float`), `address` (`string`).